### DISCUSSION TITLE
How to change selected text colour in a dialog.

### DISCUSSION THREAD

**Discussion by subbaan** [themix-project/themix-gui#513] [Q&A]
When I select text in a dialog there is no indication it is selected, to be clear I mean on the right of the Themix window where it says "Text entry" for example, or a browser address bar.
I'm on an XFCE desktop on Manjaro, I've been using a few Oomox themes I've tweaked nicely, but this one thing has me utterly stumped and because I do a bit of coding it's become quite the nuisance to not be able to see what is selected.

Edit: I don't know what I did but after 3 days on and off trying to figure this out, selected text now hilights in other apps (but still not in the "Text entry" box.  I'd still love to know how to do this as I've no idea what I fiddled with that fixed this in the couple hours since posting.

Thanks.

**Comment by actionless** [comment]
this is repository of the GUI app itself, not particular theme, create an issue in the repo of particular theme where is problem happens
